install/kubernetes/helm/istio/README.md

```diff
@@ -79,6 +79,7 @@ Helm charts expose configuration options which are currently in alpha.  The curr
 | `global.hub` | Specifies the HUB for most images used by Istio | registry/namespace | `docker.io/istio` |
 | `global.tag` | Specifies the TAG for most images used by Istio | valid image tag | `0.8.latest` |
 | `global.proxy.image` | Specifies the proxy image name | valid proxy name | `proxyv2` |
+| `global.proxy.concurrency` | Specifies the number of proxy worker threads | number | unset |
```

Comment:
Can you add a little more detail about the default, e.g. "if no value is provided, by default the proxy's concurrency will match the number of CPUs on the system" or similar? (To give users a frame of reference when configuring this.)

Reply:
I've added `0 = auto` (or would you prefer `0 = number of CPU cores`?) and more comments in the `.yaml` files, to avoid the confusing `unset`.

Let me know if that works for you.